src/components/Menu/CompositeMenu/index.js

```diff
@@ -29,49 +29,70 @@ export default class Menu extends React.Component {
     es6ClassBindAll(this);
 
     this.state = {
-      filteredItems: undefined,
+      isOpen: false,
     };
   }
 
-  onFilterItems(filteredItems) {
-    this.setState({ filteredItems });
+  componentDidMount() {
+    mediaQueryList.addListener(this.onMediaQueryListChange);
   }
 
-  renderMenu() {
-    const { items } = this.props;
-    const { filteredItems } = this.state;
+  componentWillUnmount() {
+    mediaQueryList.removeListener(this.onMediaQueryListChange);
+  }
 
-    if (!filteredItems) {
-      return (
-        <TreeMenu items={items} />
-      );
-    }
+  onMediaQueryListChange() {
+    this.closeMenu();
+  }
 
-    if (_.isEmpty(filteredItems)) {
-      return (
-        <Alert color="secondary">검색 결과가 없습니다.</Alert>
-      );
-    }
+  openMenu() {
+    this.setState({ isOpen: true });
+    lockRootScroll(true);
```

Comment:
On small size screen, this locks body's scroll when the menu is open. But this may cause a problem when users do not apply appropriate CSS as suggested in `example/src/styles.css`. Do we need to force users to apply these specific CSS or provide some options to turn off this?

Reply:
Will there be a case that users need to turn this off?

Reply:
Maybe it's not.. So it seems like it's OK to implicitly manipulate CSS including layout & scroll behavior until any customization is needed from outside.

Reply:
@krazijames I have no idea what the problem is exactly, but I think zero option is better to start with.

Reply:
@namenu Without restricting menu's height (e.g. `max-height: 100%`) the menu's height will grow by its content size and no scroll will be available including body scroll. Anyway, this problem will not appear if we manipulate CSS directly.